Dos submission questions

allait opened the pull request:
Adds a full submission manifest and question definitions for Digital Outcomes and Specialists:
* a full list of questions with correct question names, types and lot dependencies
* a full list of submission manifest sections will all questions listed in the manifest
* most of the current option values for checkbox and radios questions
* preliminary `fields:` note for pricing questions
* a script to regenerate the common specialists questions if the common options change in the future

This doesn't include:
* any of the question hints
* most of the question validators (some question have `answer_required` validators)

![screen shot 2015-11-06 at 16 33 30](https://cloud.githubusercontent.com/assets/246664/11002371/3fe6fc9a-84a4-11e5-8719-3bb87c08fdc2.png)
![screen shot 2015-11-06 at 16 33 41](https://cloud.githubusercontent.com/assets/246664/11002382/49975a0a-84a4-11e5-812a-4ea4305acf50.png)
![screen shot 2015-11-06 at 16 34 06](https://cloud.githubusercontent.com/assets/246664/11002386/4e3bbf06-84a4-11e5-8cfa-2500ddc52f07.png)
![screen shot 2015-11-06 at 16 34 15](https://cloud.githubusercontent.com/assets/246664/11002389/51f5549a-84a4-11e5-9c82-f1ebeba97176.png)

quis commented on scripts/generate_dos_specialists_questions.sh:
```diff
@@ -0,0 +1,54 @@
+make_questions "service manager" "serviceManager"
+make_questions "technical architect" "technicalArchitect"
+make_questions "user researcher" "userResearcher"
+make_questions "user researcher" "webOperations"
```
So similar! https://github.com/alphagov/digitalmarketplace-prototype/blob/master/specialists.sh

quis commented:
This PR should include a [VERSION.txt bump](https://github.com/alphagov/digitalmarketplace-frameworks/blob/dos-submission-questions/VERSION.txt)

ralph-hawkins commented:
* Cyber security **specialist** should be Cyber security **consultant**
* Each of the **Essentials** sections should be titled **[Lot name] essentials**
* User researcher appears twice in Outcomes
* Is there a way of not having the url appear in the question on this page and work as a link on the question page? — eg the q4 in Outcomes essentials
* Studio name question should be **What is the name of the lab?** not **studio**
* 'How much do you charge per day for **a** agile coach?' should be 'How much do you charge per day for **an** agile coach?'

allait commented:
@ralph-hawkins I've split Essentials into "User research participants essentials" and "Digital outcomes and specialists essentials" – since outcomes and specialists share questions there's no easy way to set different section names for them.

I've changed the question names to use markdown for URLs, but we're not applying the markdown filter to the question names in the frontend, that should be a separate story.

Fixed everything else.

ralph-hawkins commented:
@allait Can you call the Essentials questions for Outcomes and Specialists lot ‘Service essentials’ please?

allait commented:
@ralph-hawkins renamed to "Service essentials" https://github.com/alphagov/digitalmarketplace-frameworks/commit/526dd19d8063f50a80df77310575622f4c896210

ralph-hawkins commented:
:+1: :-)
